core/src/main/scala/caliban/GraphQLRequest.scala

```diff
@@ -47,6 +45,13 @@ object GraphQLRequest {
   implicit def playJsonWrites[F[_]: IsPlayJsonWrites]: F[GraphQLRequest] =
     caliban.interop.play.json.GraphQLRequestPlayJson.graphQLRequestWrites.asInstanceOf[F[GraphQLRequest]]
 
+  @deprecated("kept for compatibility purposes only", "1.7.2")
```

Comment:
Maybe next release will be 1.8.0? We have a bunch of breaking changes pending to be merged. Maybe we wait for the `oneof` support?

Reply:
To be honest if the next release is a major we can just delete these methods.

Let's merge it in as-is and I'll create another PR to delete the methods and merge it when we're certain we don't need to create a patch release